libs/web/identity/ui/src/lib/identity-ui-wallet-list.tsx

```diff
@@ -0,0 +1,101 @@
+import { ellipsify, Identity } from '@deanslist-platform/sdk'
+import { CoreUiDebugModal } from '@deanslist-platform/web-core-ui'
+import { ActionIcon, Badge, Box, Group, Image, Menu, Paper, Text } from '@mantine/core'
+import { UiGroup, UiStack } from '@pubkey-ui/core'
+import { IconDotsVertical, IconExternalLink, IconStarFilled, IconTrash } from '@tabler/icons-react'
+import { IdentityUiSolanaVerifyButton } from './identity-ui-solana-verify-button'
+import { IdentityUiVerified } from './identity-ui-verified'
+
+export function IdentityUiWalletList({
+  deleteIdentity,
+  items,
+  refresh,
+  setPrimary,
+}: {
+  deleteIdentity?: (id: string) => void
+  items: Identity[]
+  refresh?: () => void
+  setPrimary: (id: string) => void
+}) {
+  return (
+    <UiStack>
+      {items?.map((item) => (
+        <Paper withBorder key={item.id} bg="transparent" radius="md" p="sm">
+          <Group justify="space-between">
+            <Group>
+              <Box
```

Comment:
Would an <Avatar/> work better here?

Reply:
It didn't work well as it would crop the Solana logo. It has probably something to do with the dimensions of the image.